code/style.css

```diff
@@ -1,7 +1,215 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+@import url('https://fonts.googleapis.com/css2?family=Nunito');
 
-.active + .answer {
-  /* some styling here */
+*, *:before, *:after {
+  box-sizing: border-box;
 }
+
+body {
+  font-family: 'Nunito', sans-serif;
+  margin: 0 auto;
+}
+
+html {
+  scroll-behavior: smooth;
```

Comment:
Right?! I love it! When it's in `html` it applies to the whole viewport. On smaller elements it can instead almost act as a transition function (e.g. click-navigating within an iframe) or a typical 'Back to top' button using CSS. 👍